packages/react-components/src/templates/GroupProfileAbout.tsx

```diff
@@ -34,7 +35,7 @@ const GroupProfileAbout: React.FC<GroupProfileAboutProps> = ({
 }) => (
   <div css={styles}>
     <GroupInformation tags={tags} description={description} />
-    <GroupTools tools={tools} />
+    <GroupTools calendar={calendars[0] && calendars[0].id} tools={tools} />
```

Comment:
oof. can we ask design for a better spec? :/ I don't wanna get into a situation where we need to clean up 5 places in the existing product again. Multiple calendars per group isn't even a hypothetical case is it

Reply:
The calendar is a  group property and we need to surface it here. You are right that multiple calendars per group isn't a possible case, however we modelled the relationship in a way that allows for that :disappointed:.

We can create a follow up piece of work to address that.

Reply:
I was actually thinking it *is* a possible case, but okay if it's not can we put in a Squidex relationship maximum restriction, and in the backend extract the single item out of the array (throwing if it has more than one, which should not happen because of the squidex restriction)?

Reply:
I'm working on this https://trello.com/c/n0bXwLtY/973-ensure-a-group-as-only-one-calendar-and-change-the-models-to-reflect-it next to address it.